src/Computaria/Tardis/Factory.php

```diff
@@ -26,7 +26,7 @@ public static function grow(
 
     private static function createProxyFactoryIfNecessary(ProxyManager\AbstractBaseFactory $proxyFactory=null)
     {
-        if (false === is_null($proxyFactory)) {
+        if (!is_null($proxyFactory)) {
```

Comment:
@guilhermesiani  Yes. `is_null` is more lazy than `null === $var`. function calls is more lazy then operators usage.